books/free-programming-books-es.md

```diff
@@ -216,13 +216,17 @@
 
 #### AngularJS
 
-> :information_source: Véase también &#8230; [Angular](#angular)
-
 * [AngularJS](https://eladrodriguez.gitbooks.io/angularjs) - Elad Rodriguez (HTML)
+* [Angular 2](https://rldona.gitbooks.io/angular-2-book) - Raúl López (HTML)
```

Comment:
401 - not free

Reply:
The link was removed since the book was not free to access